event-stream-processing/src/__snapshots__/node-css-parser.test.ts.snap

```diff
@@ -0,0 +1,74 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`basic - GET request: 2f015294-45f8-4235-949b-667236bea641 1`] = `
+Object {
+  "@timestamp": "2021-07-14T23:17:05.036Z",
+  "agent": Object {
+    "type": "fluentbit",
+  },
+  "azp": "chefs-frontend",
+  "client": Object {
+    "ip": "2001:db8:3333:4444:5555:6666:7777:8888",
+  },
+  "event": Object {
+    "dataset": "node.css.http",
+    "kind": "event",
+    "module": "node.css",
+    "original": Object {
+      "azp": "chefs-frontend",
+      "clientIp": "2001:db8:3333:4444:5555:6666:7777:8888",
+      "contentLength": "323",
+      "hostIp": "::ffff: 10.97.8.1",
+      "httpVersion": "1.1",
+      "level": "http",
+      "message": "GET /api/v1/forms?active=true 200 624ms",
+      "method": "GET",
+      "path": "/api/v1/forms",
+      "query": Object {
+        "active": "true",
+        "deleted": "false",
+      },
+      "responseTime": 624,
+      "statusCode": 200,
+      "timestamp": "2021-07-14T23: 17: 05.036Z",
+      "userAgent": "Mozilla/5.0",
+    },
+    "type": "http",
+  },
+  "host": Object {
```

Comment:
i think i follow.
i will remap the openshift pod details to:
`host.pod_name` and `host.namespace.name`

Reply:
in this case, can yo use `kubernetes` instead of `host` as listed above. I don't think we will get much more metadata about the namespace, so keep as 1 field (with underscore)

Reply:
@TimCsaky , isn't `host.tip` now `kubernetes.pod_id`? or is that the underline node IP where the pod is running?

Reply:
@cvarjao this would be the ip address of the node IP where the pod is running.

Reply:
we could set this to a `kubernetes.` field if you like(?)

Reply:
Is it though? if it is the underline node IP then yes. `host.ip` is correct. However, if it is the pod IP, then it is `kubernetes_pod_id`. Those are different things.

Reply:
done